domains/naffy.json

```
@@ -1,9 +1,9 @@
 {
   "owner": {
     "username": "knownasnaffy",
-    "twitter": "dharni_naffy"
+    "email": "contact@barinderpreet.com"
   },
   "records": {
-    "CNAME": "knownasnaffy.github.io"
+    "CNAME": "edge.redirect.pizza"
```

```suggestion
    "URL": "https://barinderpreet.com"
```

We have a built-in redirect service which you can use for free, if you'd like, so you don't have to use any external services.

---

Great! Wonder how I missed it in the docs earlier. Thank you.